solution02/src/test/jmeter/02-Soulution-02-Google.jmx

```diff
@@ -282,8 +282,8 @@
             <collectionProp name="AuthManager.auth_list">
               <elementProp name="" elementType="Authorization">
                 <stringProp name="Authorization.url"></stringProp>
-                <stringProp name="Authorization.username">${Gusername}</stringProp>
-                <stringProp name="Authorization.password">${Gpassword}</stringProp>
+                <stringProp name="Authorization.username">admin</stringProp>
+                <stringProp name="Authorization.password">admin</stringProp>
```

Comment:
Passwords need to parameterized or kept blank? We need to document it and mention this in the README.md

Reply:
Fix the issue by parameterizing 
https://github.com/wso2-incubator/identity-test-integration/pull/145